src/community/verto_contract_oracle.ts

```diff
@@ -9,8 +9,8 @@ import { ContractReader } from './contract_oracle';
 export class VertoContractReader implements ContractReader {
 	/** Fetches smartweave contracts from the Verto cache */
 	public async readContract(txId: TransactionID): Promise<unknown> {
-		const response: AxiosResponse = await axios.get(`https://v2.cache.verto.exchange/${txId}`);
-		const contract = response.data;
+		const response = await fetch(`https://v2.cache.verto.exchange/${txId}`);
+		const contract = await response.json();
```

Comment:
Not sure what you ran into with axios here, but it doesn't really matter. I only recommend axios because thats what arweave-js already bundles with